reaktive/src/commonMain/kotlin/com/badoo/reaktive/maybe/Merge.kt

```diff
@@ -41,13 +48,9 @@ fun <T> Iterable<Maybe<T>>.merge(): Observable<T> =
                 }
             )
         }
-
-        if (activeSourceCount.addAndGet(-1) == 0) {
```

Comment:
Same here. Could please adopt and add the provided test above for this operator?

Reply:
Done.